shared/engine/session.js

```diff
@@ -115,15 +115,15 @@ class Session {
       sessionID: this.id,
     }
 
-    rpcLog('engineInternal', 'session start call', this.id, method, this)
+    rpcLog('engineInternal', 'session start call', {id: this.id, method, this: this})
     const outgoingRequest = new OutgoingRequest(method, wrappedParam, wrappedCallback, this._makeWaitingHandler(true, method), this._invoke)
     this._outgoingRequests.push(outgoingRequest)
     outgoingRequest.send()
   }
 
   // We have an incoming call tied to a sessionID, called only by engine
   incomingCall (method: MethodKey, param: Object, response: ?Object): boolean {
-    rpcLog('engineInternal', 'session incoming call', this.id, method, this, response)
+    rpcLog('engineInternal', 'session incoming call', {id: this.id, method, this: this, response})
```

Comment:
Yep! It works 
<img width="983" alt="screen shot 2016-09-06 at 1 06 50 pm" src="https://cloud.githubusercontent.com/assets/16893/18289086/d2e5af5e-7432-11e6-829e-fadb9376f425.png">